### PR TITLE
docs: supabase migration guide

### PR DESCRIPTION
**Pull request by sixhobbits**

### CODE REVIEW

File: site/docs/v1/tech/migration-guide/supabase.adoc [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+* Projects are a high-level construct that groups entities such as users and applications together. FusionAuth calls these Tenants.
+* Supabase has other products your applications might be using, like Edge functions. FusionAuth does not have equivalents for these products, and is focused only on authentication.
+
+==== Identifiers
+
+When creating an object with the FusionAuth API, you can specify the Id. It must be a link:/docs/v1/tech/reference/data-types#uuids[UUID].
+
+This works for users, applications, and tenants, among others.
+
+Supabase uses a custom UUID format. When migrating, a new UUID will be created for the user on FusionAuth, or you can use the Supabase user Id migration.
```
Comment (mooreds) [inline site/docs/v1/tech/migration-guide/supabase.adoc:71]: What is the Supabase user Id migration? I am not familiar with that.

File: site/docs/v1/tech/migration-guide/supabase.adoc [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+1. Access the Supabase SQL Editor and create a new query.
+2. Paste the `ufn_get_user_migration_data_json()` function code into the editor.
+3. Run the function in the editor and save the JSON data to a file, for example, `users.json`.
+
+The script below is a user-defined psql function for exporting user data from Supabase. The script will generate a JSON array object.
+
+[source,shell,title="Supabase user data export function"]
+----
+include::https://raw.githubusercontent.com/ritza-co/fusionauth-import-scripts/supabase/supabase/supabase_user_migration_script.sql[]
```
Comment (mooreds) [inline site/docs/v1/tech/migration-guide/supabase.adoc:87]: Please open a PR so I can merge this into fusionauth/fusionauth-import-scripts.

File: site/docs/v1/tech/migration-guide/supabase.adoc [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+
+include::docs/v1/tech/migration-guide/_create-test-application.adoc[]
+
+==== Add an API Key
+
+include::docs/v1/tech/migration-guide/_create-api-key-social.adoc[]
+
+=== Get the Script
+
+include::docs/v1/tech/migration-guide/_get-script.adoc[]
```
Comment (mooreds) [inline site/docs/v1/tech/migration-guide/supabase.adoc:183]: https://github.com/FusionAuth/fusionauth-import-scripts doesn't currently include a `supabase` directory.

**Review by mooreds — CHANGES_REQUESTED** [review]
A few small changes.

File: site/docs/v1/tech/migration-guide/supabase.adoc (hunk 2 above)
Comment (sixhobbits) [reply site/docs/v1/tech/migration-guide/supabase.adoc:87]: oops, here it is now https://github.com/FusionAuth/fusionauth-import-scripts/pull/11

File: site/docs/v1/tech/migration-guide/supabase.adoc (hunk 1 above)
Comment (rideam) [reply site/docs/v1/tech/migration-guide/supabase.adoc]: That was an error. I have removed that bit.

**Comment by rideam** [comment]
the asciidoc_check should work fine once the PR to the import scripts is merged (a linked include for sql script is not yet merged).

**Comment by mooreds** [comment]
Closing under https://github.com/FusionAuth/fusionauth-site/pull/2406 because there was one small change on the include file I needed to make.

**Pull request closed without merging**